tests/Generator.Tests/Snapshots/IProgressTests.PreserveIProgressExpressionArgument_n=01#CallWithIProgressAsync.g.verified.cs

```diff
@@ -1,6 +1,6 @@
-﻿//HintName: Test.Class.CallWithIProgressAsync.g.cs
-public static void CallWithIProgress()
-{
-    var progress = new global::System.Progress<float>();
-    WithIProgress(progress);
-}
+﻿//HintName: Test.Class.CallWithIProgressAsync.g.cs

```

Comment:
Ah.. I had the issue again with line endings. I've disabled `autoclrf` in Git and replaced all line endings, but it seems were also wrong.

Should I revert these changes? Or is it OK that they are in this PR?